Sources/TuistLoader/Models+ManifestMappers/TargetAction+ManifestMapper.swift

```diff
@@ -48,18 +48,7 @@ extension TuistGraph.TargetScript {
             runForInstallBuildsOnly: runForInstallBuildsOnly,
             shellPath: shellPath
         )
-    }
-
-    private static func absolutePaths(for paths: [Path], generatorPaths: GeneratorPaths) throws -> [AbsolutePath] {
```

Comment:
This allowed supporting both glob patterns as well as raw strings that contained variables.

As an example, if the fixture in https://github.com/tuist/tuist/tree/main/projects/tuist/fixtures/ios_app_with_actions were to be modified to include

```swift
                .pre(
                    tool: "/bin/echo",
                    arguments: ["\"tuist\""],
                    name: "Tuist",
                    inputPaths: [
                        "Sources/**/*.swift",
                        "$(SRCROOT)/Sources/AppDelegate.swift"
                    ]
                ),
```

And running the latest tuist release we get the correct results in the generated project:

```diff
  inputPaths = (
    Sources/AppDelegate.swift,
+   "$(SRCROOT)/Sources/AppDelegate.swift",
  );
```